static/files/legacy/hoprd-docker.cfg.yaml

```
@@ -0,0 +1,16 @@
+    strategies:
+      - !Aggregating
+        aggregation_threshold: 100
+        unrealized_balance_ratio: 0.9
+        aggregation_timeout: 60
+        aggregate_on_channel_close: true
+      - !AutoRedeeming
+        redeem_only_aggregated: true
+        minimum_redeem_ticket_value: "30000000000000000000 HOPR"
+        on_close_redeem_single_tickets_value_min: "90000000000000000 HOPR"
+      - !ClosureFinalizer
+        max_closure_overdue: 3600
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)